bofire/data_models/surrogates/mlp.py

```diff
@@ -20,3 +20,4 @@ class MLPEnsemble(BotorchSurrogate, TrainableSurrogate):
     subsample_fraction: Annotated[float, Field(gt=0.0)] = 1.0
     shuffle: bool = True
     scaler: ScalerEnum = ScalerEnum.NORMALIZE
+    output_scaler: ScalerEnum = ScalerEnum.STANDARDIZE
```

Comment:
We should add a validator here, that blocks the `Normalize` enum, so that it crashes when creating the data model and not when training it.

Reply:
Added the validator